tools/clang/test/HLSLFileCheck/hlsl/functions/attribute/attributeAddition.hlsl

```diff
@@ -0,0 +1,10 @@
+// RUN: %dxc -T cs_6_6 -E mymain -ast-dump %s | FileCheck %s
+
+// CHECK: mymain 'void ()'
+// CHECK: -HLSLShaderAttr
+// CHECK-SAME: "compute"
+
+[numthreads(1,1,1)]
+void mymain(){
```

Comment:
We probably need some more tests for entry functions, such as ones that should be ignored, like in namespaces, classes, or inside a function, and declarations vs. definitions.